app/services/publish_content_item.rb

```diff
@@ -0,0 +1,34 @@
+class PublishContentItem
```

Comment:
I've reverted this to its previous behaviour now that we use the Sidekiq worker for services and information pages